### PR TITLE
Add support for ZRANDMEMBER command

### PR DESCRIPTION
**Pull request by dengliming**
Resolves [https://github.com/redis/jedis/issues/2447](https://github.com/redis/jedis/issues/2447)

### CODE REVIEW

File: src/test/java/redis/clients/jedis/tests/commands/SortedSetCommandsTest.java [hunk 1]
```diff
@@ -1445,4 +1444,47 @@ public void bzpopmin() {
     actual = jedis.bzpopmin(0, bbar, bfoo);
     assertEquals(new KeyedTuple(bbar, bc, 0.1d), actual);
   }
+
+  @Test
+  public void zrandmember() {
+    Map<String, Double> hash = new HashMap<>();
```
Comment (sazzad16) [inline src/test/java/redis/clients/jedis/tests/commands/SortedSetCommandsTest.java:1450]:
```suggestion
  public void zrandmember() {

    assertNull(jedis.zrandmember("foo"));
    assertNotNull(jedis.zrandmember("foo", 1));
    assertNotNull(jedis.zrandmemberWithScores("foo", 1));

    Map<String, Double> hash = new HashMap<>();
```

**Review by sazzad16 — CHANGES_REQUESTED** [review]

File: src/main/java/redis/clients/jedis/BinaryJedis.java [hunk 2]
```diff
@@ -2865,7 +2886,7 @@ public Long zcount(final byte[] key, final byte[] min, final byte[] max) {
 
   protected Set<Tuple> getTupledSet() {
     List<byte[]> membersWithScores = client.getBinaryMultiBulkReply();
-    if (membersWithScores.isEmpty()) {
+    if (membersWithScores == null || membersWithScores.isEmpty()) {
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/BinaryJedis.java:2889]: For which case are you getting `membersWithScores == null`?

Comment (dengliming) [reply src/main/java/redis/clients/jedis/BinaryJedis.java]: When the key was not exist, it will throw NPE.

Comment (sazzad16) [inline src/main/java/redis/clients/jedis/BinaryJedis.java:2889]: If response from Redis `nil`, we should return `null`.

**Review by sazzad16 — CHANGES_REQUESTED** [review]

File: src/main/java/redis/clients/jedis/BinaryJedis.java (hunk 2 above)
Comment (dengliming) [reply src/main/java/redis/clients/jedis/BinaryJedis.java]: OK

Comment (sazzad16) [reply src/main/java/redis/clients/jedis/BinaryJedis.java:2889]: return `null` in that particular case.

**Review by sazzad16 — APPROVED** [review]

**Pull request merged**